models/api-models.js

```diff
@@ -0,0 +1,7 @@
+const endpoints = require('../endpoints.json')
+
```

Comment:
You wouldn't necessarily need a model for this. You could read the file in the controller as you're not interacting with any databases